pkg/virt-handler/vm_test.go

```diff
@@ -3077,6 +3081,12 @@ var _ = Describe("VirtualMachineInstance", func() {
 	})
 })
 
+func markCompletedNetworkSetup(controller *VirtualMachineController, vmi *v1.VirtualMachineInstance) {
```

Comment:
Would you be open to renaming this func to something with `mock` in it ? I inferred what it did because it's a function that doesn't return anything, but still ...

Reply:
Thus injecting a mocked behavior (no-op) into the real object.